### PR TITLE
Search for non string queries

### PR DESCRIPTION
**Pull request by Delerme**

### CODE REVIEW

File: src/ui/components/search/searchcomponent.js [hunk 1]
```diff
@@ -131,7 +131,7 @@ export default class SearchComponent extends Component {
   }
 
   onCreate () {
-    if (this.query && this.query.length > 0) {
+    if (this.query) {
```
Comment (benmcginnis) [inline src/ui/components/search/searchcomponent.js:134]: well that's interesting, man

Comment (benmcginnis) [reply src/ui/components/search/searchcomponent.js:134]: what about `00000` i feel like that's probably a valid zip somewhere

Comment (Delerme) [reply src/ui/components/search/searchcomponent.js:134]: `00000` worked for me, I think it gets coerced into a string, but admittedly `0` doesn't

Comment (Delerme) [reply src/ui/components/search/searchcomponent.js:134]: ended up just `String()`ing everything

**Review by b1lly — APPROVED** [review]

**Pull request merged**